lib/assessment.rb

```diff
@@ -42,7 +46,10 @@ def transform(words)
   # named filename. Concatenate the lines together after
   # replacing the trailing newline character with a
   # space and return the result.
-  def read_file(filename)
-    # Your code goes here
-  end
+def read_file(filename)
+    File.open(ARGV[0]) do |file|
+
+      file.reduce('') {|memo, item| memo + item.chop + ' '}
```

Comment:
Mostly correct, but don't forget to remove the trailing space! One tool for that is `strip`.